app/components/view-epics/macro.test.js

```diff
@@ -0,0 +1,458 @@
+        request(dummyApp)
+          .get('/')
+          .then((res) => {
+            const $ = cheerio.load(res.text);
+            expect($('[data-test-id="some-data-identifier"] .must-not-met-epics').length).toEqual(0);
+
+            done();
+          });
+      });
+    });
+  });
+
+  describe('for may epics', () => {
+    it('should render the tag if data provided', (done) => {
+      const context = {
+        params: {
+          dataTestId: 'some-data-identifier',
+          data: {
+            may: {
+              met: [
+                {
+                  id: 'C14E1',
+                  name: 'access prescribable items',
+                },
+              ],
+              'not-met': [
+                {
+                  id: 'C14E13',
+                  name: 'access Patient Record',
+                },
+              ],
+            },
+          },
+        },
+      };
+
+      const dummyApp = createTestHarness(macroWrapper, context);
+      request(dummyApp)
+        .get('/')
+        .then((res) => {
+          const $ = cheerio.load(res.text);
+          expect($('[data-test-id="may-tag"]').text().trim()).toEqual('May Epics');
+
+          done();
+        });
+    });
+
+    it('should render the data if provided', (done) => {
+      const context = {
+        params: {
+          dataTestId: 'some-data-identifier',
+          data: {
+            may: {
+              met: [
+                {
+                  id: 'C14E15',
+                  name: 'view EPS Nominated Pharmacy changes',
+                },
+                {
+                  id: 'C14E16',
+                  name: 'view EPS Nominated Pharmacy changes',
+                },
+              ],
+              'not-met': [
+                {
+                  id: 'C14E17',
+                  name: 'Configure warnings for prescribable items',
+                },
+                {
+                  id: 'C14E18',
+                  name: 'maintain referal record',
+                },
+              ],
+            },
+          },
+        },
+      };
+
+      const dummyApp = createTestHarness(macroWrapper, context);
+      request(dummyApp)
+        .get('/')
+        .then((res) => {
+          const $ = cheerio.load(res.text);
+          expect($('[data-test-id="may-epics"]').length).toEqual(1);
+
+          done();
+        });
+    });
+
+    it('should not render the data if not provided', (done) => {
+      const context = {
+        params: {
+          dataTestId: 'some-data-identifier',
+          data: {},
+        },
+      };
+
+      const dummyApp = createTestHarness(macroWrapper, context);
+      request(dummyApp)
+        .get('/')
+        .then((res) => {
+          const $ = cheerio.load(res.text);
+          expect($('[data-test-id="may-epics"]').length).toEqual(0);
+
+          done();
+        });
+    });
+
+    describe('with met type', () => {
+      it('should render the data if provided', (done) => {
```

Comment:
`should render the viewEpicList component when provided with epics that are may and met`